### DISCUSSION TITLE
Where to add --upgrade

### DISCUSSION THREAD

**Discussion by west1778** [vladmandic/sdnext#627] [General]
I'm a total newb and I see that you can add that command to upgrade to make sure things are being updated but where do I find the place to add it?

**Comment by Tillerz** [comment]
webui.bat --upgrade

**Reply by west1778** [reply]
Just anywhere in there? I don't see a spot for arguments.

**Comment by Tillerz** [comment]
That is the batch file you use to start it. Just add the parameter when starting the ui. You can make your own .bat/.cmd file and call it like that.

Start_My_SD_UI.bat

```
@echo off
.\webui.bat --upgrade
```

Drop it in the same folder and click it.

Or alternatively, make a desktop link to that webui.bar and then rightclick and edit it on the desktop and add the --upgrade there to the filename.

**Reply by econundrum** [reply]
Or right click on webui.bat and choose create a shortcut. Then open the properties and after it says webui.bat in what to run add - - upgrade then click apply you can put that shortcut on your desktop and whenever you run it it will use the - - upgrade command line argument.

You can easily create a few different shortcuts if you like with different combinations of command line attributes. For instance I have on that upgrades, one that doesn't.

**Reply by west1778** [reply]
![image](https://user-images.githubusercontent.com/73608023/235350575-bf6aba90-6d86-439e-960b-80efe68c613d.png)

So add it to which line in here?

**Reply by econundrum** [reply]
The end of the target line, leave a space after webui.bat and then put - - upgrade

If you want multiple command line arguments leave a space between them.

**Reply by west1778** [reply]
ah, thanks mate!

**Reply by econundrum** [reply]
No problem, easy once you know how, but if you've never done it you won't know.

**Reply by west1778** [reply]
Yeah I was looking for the way you do it with A1111 with the adding git pull so I was totally confused lol
